core/database/src/main/java/com/yapp/database/MissionTypeConverter.kt

```
@@ -0,0 +1,21 @@
+package com.yapp.database
+
+import androidx.room.TypeConverter
+import com.yapp.domain.model.MissionType
+
+class MissionTypeConverter {
+
+    @TypeConverter
+    fun fromString(missionType: String): MissionType {
+        return return try {
+            MissionType.valueOf(missionType)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)